src/utils/read-file.ts

```
@@ -38,24 +38,10 @@ export const readConfigFile = (basePath: string, fileName: string): File => {
 export const readConfigFileAtPath = (filePath: string): File =>
   readConfigFile(path.dirname(filePath), path.basename(filePath))
 
-export const readSchemaFile = (
-  basePath: string,
-  fileName = 'schema.json'
-): File | undefined => {
-  const filePaths = findFiles(basePath, fileName)
-
-  if (R.isEmpty(filePaths)) {
+export const readSchemaFile = (schemaPath: string): File | undefined => {
+  if (R.isNil(schemaPath) || !isJson(schemaPath)) {
     return undefined
```

Should this maybe throw an error instead? Missing schema seems important.

---

We support having no schema on`load()`

---

I think `null` might be a more appropriate return here: https://airbrake.io/blog/javascript-error-handling/null-undefined-properties

---

aight

---

updated 👍